packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -133,5 +133,11 @@ class WidgetTester extends Instrumentation {
 void testWidgets(callback(WidgetTester tester)) {
   new FakeAsync().run((FakeAsync async) {
     callback(new WidgetTester._(async));
```

Comment:
for symmetry, can you move the runApp call from the Widget Tester constructor to here before the callback? (assuming it has the same semantics, I didn't check too closely)